File: ExampleApp/app/src/main/java/com/motiv/example/BottomNavigationDashboardContract.java
package com.motiv.example;

import androidx.fragment.app.*;
import dagger.*;
import dagger.android.*;
import dagger.android.support.*;
import javax.inject.*;

public interface BottomNavigationDashboardContract {

    public interface View {

        public void navigationControllerstartDestinationById(int arg0);
    }

    public interface Presenter {

        public void eloonNavigationItemSelected(android.view.MenuItem argument0);
    }
}

File: ExampleApp/app/src/main/java/com/motiv/example/BottomNavigationDashboardContract.kt
 
package com.motiv.example

public interface BottomNavigationDashboardContract {

    public interface View {

        public fun navigationControllerstartDestinationById(arg0: Int): Unit
    }

    public interface Presenter {

        public fun eloonNavigationItemSelected(argument0: android.view.MenuItem): Unit
    }
}
